library/x509write_crt.c

```diff
@@ -293,6 +293,44 @@ int mbedtls_x509write_crt_set_key_usage( mbedtls_x509write_cert *ctx,
     return( 0 );
 }
 
+int mbedtls_x509write_crt_set_ext_key_usage( mbedtls_x509write_cert *ctx,
+                                             const mbedtls_asn1_sequence *exts )
+{
+    unsigned char buf[256];
+    unsigned char *c = buf + sizeof(buf);
+    int ret;
+    size_t len = 0;
+    const mbedtls_asn1_sequence *last_ext = 0, *ext;
```

Comment:
use `NULL` rather than `0` for a NULL pointer.

generally, when there's assignments like this, and non-simple types, it's clearer to have different variables declared on different lines

```suggestion
    const mbedtls_asn1_sequence *last_ext = NULL;
    const mbedtls_asn1_sequence *ext;
```